apps/internal/services/e2e_setup.py

```
@@ -0,0 +1,239 @@
+import logging
+import os
+import uuid
+
+logger = logging.getLogger(__name__)
+logger.level = logging.INFO
+
+
+class E2ESetupService:
+    """Service for setting up E2E test fixture data"""
+
+    def __init__(self, admin_email, user_id, refresh_token, org_id, cluster_domain):
```

Pylint (3.3.7)

[refactor] Too many positional arguments (6/5)

(R0917)